__tests__/app.test.js

```diff
@@ -25,3 +25,31 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topics, each having 'slug' and 'description'", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toBeInstanceOf(Array);
+        topics.forEach((topic) => {
```

Comment:
This test will still pass even if we get back an empty array - why? 
To mitigate this - any time you use a forEach in your test to assert something, you need to check that the array has the right length - outside of the forEach